content/components/action-bar.mdx

```diff
@@ -0,0 +1,277 @@
+---
+title: Action bar
+status: Experimental
+---
+
+import {Box, Label, LabelGroup, Text} from '@primer/react'
+
+<Box sx={{fontSize: 3}} class="lead" as="p">
+  An action bar contains a collection of icon buttons placed horizontally.
+</Box>
+
+<!-- TODO: uncomment when available
+<LabelGroup>
+  <Label
+    as="a"
+    href="https://primer.style/react/ActionBar"
+    size="large"
+    variant="secondary"
+    sx={{textDecoration: 'none'}}
+  >
+    <img
+      width="16"
+      height="16"
+      alt=""
+      src="https://user-images.githubusercontent.com/293280/123878374-ce9d4d00-d8f3-11eb-8adf-1a160292ff53.png"
+      style="vertical-align: middle; margin-right: 4px;"
+    />
+    React
+  </Label>
+  <Label
+    as="a"
+    href=""
+    size="large"
+    variant="secondary"
+    sx={{textDecoration: 'none'}}
+  >
+    <img
+      width="16"
+      height="16"
+      alt=""
+      src="https://user-images.githubusercontent.com/293280/123878720-80d51480-d8f4-11eb-9b10-d02a1cb606f8.png"
+      style="vertical-align: middle; margin-right: 4px;"
+    />
+    Figma
+  </Label>
+  <Label
+    as="a"
+    href="https://primer.style/view-components/components/alpha/actionbar"
+    size="large"
+    variant="secondary"
+    sx={{textDecoration: 'none'}}
+  >
+    <img
+      width="16"
+      height="16"
+      alt=""
+      src="https://user-images.githubusercontent.com/18661030/171468311-184bfd70-128e-4bfc-b482-96d0de49d42a.png"
+      style="vertical-align: middle; margin-right: 4px;"
+    />
+    Rails
+  </Label>
+</LabelGroup>
+-->
+
+## Overview
+
+Use an action bar to render multiple icon buttons in a row. Buttons can be split into groups by adding a divider. When there is not enough space, buttons that don't fit will be added to an overflow menu.
+
+## Anatomy
+
+<img width="960" alt="A diagram of an action bar with a few buttons, a divider and at the end a button that opens an overflow menu." src="https://user-images.githubusercontent.com/378023/188037013-e0c071ee-19f5-4a1c-9deb-3bc695d71cec.png" />
+
+## Content
+
+### Buttons
+
+An action bar should only contain icon buttons with the `invisible` variant (no border/background).
+
+<DoDontContainer>
+  <Do>
+    <img
```

Comment:
For all the images with `<Caption>` make sure you add `alt=""` to the `img` tag

Reply:
Oh, I see `role="presentation"` 🤔 I'm not sure which is better (or if both are needed) but we usually just use an empty alt and that hasn't been flagged by anyone to my knowledge.

Reply:
Hmm.. yeah, not sure either. I think I used the `SegmentedControl` as a template. @mperrotti is using `role="presentation"` preferred over `alt=""` or are both options ok?